.github/workflows/windows-package.yml

```diff
@@ -0,0 +1,36 @@
+# This workflow will install compilers and packges on windows platform
+
+name: windows package
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master, develop ]
+
+name: windows
+jobs:
+  windows:
+    runs-on: windows-2019
+    strategy:
+      matrix:
+        compiler: [gcc, gcc-fortran]
```

Comment:
omit this, we need both compilers to be installed, not two different runs with different compilers